lib/datadog/profiling/ext/forking.rb

```diff
@@ -45,15 +45,15 @@ def fork
             child_block = if block_given?
                             proc do
                               # Trigger :child callback
-                              at_fork_blocks[:child].each(&:call) if at_fork_blocks.key?(:child)
+                              ddtrace_at_fork_blocks[:child].each(&:call) if ddtrace_at_fork_blocks.key?(:child)
```

Comment:
Should this be namespaced as `datadog_` instead of `ddtrace_`?

Reply:
I like using `ddtrace` here since matches the gem name, but this is a weakly-held opinion :)